eng/pipelines/runtimelab.yml

```diff
@@ -100,88 +100,3 @@ jobs:
       nameSuffix: All_Configurations
       buildArgs: -s clr.runtime+libs -c $(_BuildConfig) -allConfigurations
 
-#
```

Comment:
@joperezr The CI infrastructure in the NativeAOT branch is going to be a bit different from how CI is done in the runtime repo (we can't do AOT compilation on the Helix machines because they miss some big prerequisites, so we'll do the testing on the build machine - this should be similar how the current plan for iOS testing in the runtime repo will be).

Do you think it makes sense to try to share the runtimelab.yml file, or just make a new one to avoid constant merge conflicts?

Reply:
It’s ok to have another yml file if desired, but that will mean that we have to setup a whole new pipeline on AzDO for this branch so just keep thag in mind, since AzDO pipelines are tied to only one yml file. On the other hand, if you do decide to change runtimelab.yml to suit your needs, we could just ignore conflicts from that file and keep branch’s version always.

Reply:
FWIW, you shouldn’t expect to get many conflicts as the idea is to have runtimelab.yml as stable as possible, which in fact was the whole reason we created that as opposed to using runtime.yml which gets changed much more frequently.

Reply:
Thanks! I didn't know that extra yaml file would be work - I'll just keep this and we'll ignore conflicts in the future integrations.